app/functions.R

```diff
@@ -120,44 +119,46 @@ renderScoreHtml <- function(risk) {
-                     vaccine_labels[vaccine], " COVID-19 vaccine!",
-                     "This vaccine reaches its full ", formatPercent(vaccines[[vaccine]]$efficacy),
-                     " efficacy at around ", vaccines[[vaccine]]$days_after_final_dose,
-                     ifelse(vaccines[[vaccine]]$doses==1, "days after vaccination. ", "days after the second dose. "), 
-                     "Your immunity will build up over the next few days. ",
-                     tags$a("Click here ", href=urls$cdc_vaccines), 
-                     "for more information about the United States' vaccination program.")
-    } else {
-      # after efficacy threshold
-      text <- tags$p("Congratulations on receiving", 
-                     ifelse(vaccines[[vaccine]]$doses==1, "the ", "both doses of the "), 
-                     vaccine_labels[vaccine], " COVID-19 vaccine!",
-                     "This vaccine reduces your risk of contracting symptomatic COVID-19 by ",
-                     formatPercent(vaccines[[vaccine]]$efficacy), ". ",
-                     tags$a("Click here ", href=urls$cdc_vaccines), 
-                     "for more information about the United States' vaccination program.")
+  } else if (!is.na(months_last_vaccination)) {
+    # if has vaccination
+    infection_efficacy_perc <- formatPercent(risk$vaccine_reduction$infection_efficacy)
+    hosp_efficacy_perc <- formatPercent(risk$vaccine_reduction$hosp_efficacy)
+    icu_efficacy_perc <- formatPercent(risk$vaccine_reduction$icu_efficacy)
+    death_efficacy_perc <- formatPercent(risk$vaccine_reduction$death_efficacy)
+    
+    if (months_last_vaccination == "lt_1mo"){
+      # last vaccine within 1 month 
+      months_last_vaccination_text <- HTML(paste0("Congratulations on receiving the ", vaccine_labels[vaccine], " COVID-19 vaccine! ", 
+                     "The vaccine reaches its full efficacy at around 14 days after vaccination. ",
+                     "We estimate that the full efficacy of the ", vaccine_labels[vaccine]))
+    } else if (months_last_vaccination == "btw_1_6mo"){
+      # last vaccine within 1-6 month
+      months_last_vaccination_text <- HTML(paste0("Congratulations on receiving the ", vaccine_labels[vaccine], " COVID-19 vaccine! ",
+                                             "Between 1 and 6 months after last vaccination, we estimate that the efficacy of the ", vaccine_labels[vaccine]))
+      
+    } else if (months_last_vaccination == "gt_6mo"){
+      # last vaccine within greater than 6 month
+      months_last_vaccination_text <- HTML(paste0("Congratulations on receiving the ", vaccine_labels[vaccine], " COVID-19 vaccine! ",
+                     "Be sure check if you are eligible to receive a booster shot. ",
+                     "More than 6 months after last vaccination, we estimate that the efficacy of the ", vaccine_labels[vaccine]))
     }
+    
+    reduction_text <- HTML(paste0( " COVID-19 vaccine is a ",  
```

Comment:
This sentence needs some wordsmithing. I'll go fire up the app now and suggest some edits.

Reply:
Update, hit an error from the API. I guess this is because we have not deployed the API. @pendlrobinson  Is there another end point that you tested against when you developed?
![image](https://user-images.githubusercontent.com/8843067/146442250-3e641f89-207f-4078-a9c5-991a2eb3d28f.png)

Reply:
Yes, I locally deployed the the calculator[ feature/309_boostershot_backend](https://github.com/mathematica-mpr/math-api-covid-risk-calculator/tree/feature/309_boostershot_backend) branch. I changed the Shiny apps global variable `covid_score_api` to point to the local API end point (http://127.0.0.1:3000/score). Then I was able to test the RShiny app (outside of the docker container).